### ISSUE TITLE
RadioGroup is not focusable if no value is initially selected

### ISSUE THREAD

**Issue by mfal** [mittwald/flow#512]
![Bildschirmfoto 2024-06-13 um 16 09 24](https://github.com/mittwald/flow/assets/4696750/6a873022-1926-44d4-8953-e88dfd75dfa1)

**Comment by Lisa18289** [comment]
hab ich gerade getestet, funktioniert jetzt
